pysatModels/tests/test_utils_extract.py

```diff
@@ -477,3 +477,101 @@ def test_alternate_output_names(self):
         assert np.all(alt_diff <= 1.0)
 
         return
+
+
+@pytest.mark.skipif(pysat.__version__ == '3.0.1',
```

Comment:
Should also skip for lower pysat versions.  Recommend creating a pysat version float variable at the top of this test.

Reply:
I did this recently (probably SpaceWeather?) Might be a useful test utility in pysat